### ISSUE TITLE
SetItemInMainHand / SetItemInOffHand - Visual glitch for surrounding players.

### ISSUE THREAD

**Issue by andrewkm** [libraryaddict/LibsDisguises#235]
I'm going to give reproduction steps as opposed to trying to explain this as it may sound a bit confusing. So, lets get right to it.

Bob and Joe are online.

1. Bob does /d vex setiteminmainhand 276
2. Bob hits f5 and does indeed see himself as a vex with a diamond sword (276) in it's hand.
3. Joe sees a vex without a diamond sword in it's hand (276)

To fix the issue

4. Bob moves his mouse wheel to a slot that has an item currently (http://i.imgur.com/oe4ywKN.png)
5. Joe now sees a vex with a diamond sword in it's hand as he should.
6. Bob can now move his mouse wheel away from an item slot as it no longer matters, Joe will always see the correct visual representation.

Same can be reproduced with setiteminoffhand. However for step 4 Bob must place an actual item in off hand for the visuals to update for Joe.

Hopefully that makes sense.

TL;DR: To fix maybe when using SetItemInMainHand / SetItemInOffHand the plugin should somehow push an update so surrounding players see what they are supposed to? I'm not sure, total guess, but yeah, that's the bug, hopefully it's possible to even fix.

**Comment by andrewkm** [comment]
And before I forget:
Using Jenkins: 9.4.0-Dev-#135 + Premium in plugins folder.

**Comment by libraryaddict** [comment]
Fixed
